src/uct/ib/base/ib_iface.c

```diff
@@ -855,9 +863,9 @@ static ucs_status_t uct_ib_iface_init_pkey(uct_ib_iface_t *iface,
         }
     }
 
+out:
```

Comment:
1. change to "out_pkey_found"
2. replace "break" in line 843 by "goto out_pkey_found"
3. replace if in line 847 by `ucs_assert(!pkey_found)`

Reply:
done